components/phoenix/combicon-mc.stanza

```diff
@@ -0,0 +1,63 @@
+#use-added-syntax(jitx)
+defpackage ocdb/phoenix/combicon-mc :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+public pcb-component component (n:Int):
+  name = "Phoenix screw terminal"
+  manufacturer = "Phoenix"
+  mpn = switch(n) :
+    2 : "1803277"
+    3 : "1803280"
+    4 : "1803293"
+    5 : "1803303"
+    6 : "1803316"
+    7 : "1803329"
+    8 : "1803332"
+    9 : "1803345"
+    10 : "1803358"
+    11 : "1803361"
+    12 : "1803374"
+    13 : "1803387"
+    14 : "1803390"
+    15 : "1803400"
+    16 : "1803413"
+    18 : "1841297"
+    20 : "1841271"
+    else : fatal("Incorrect n-pin for Phoenix Connector %_" % [n])
+    
+  pin-properties :
+    [pin:Ref     | pads:Int ...   | side:Dir]
+    for i in 1 through n do :
+      [p[i] | i | Left]
+  make-box-symbol()
+  assign-landpattern(phoenix-combicon-mc-pkg(n))
+  reference-prefix = "J"
+
+
+pcb-landpattern phoenix-combicon-mc-pkg (n:Int) :
+  val a = 3.81 * to-double(n - 1)
+  val x0 = a / -2.0
+  val y0 = 8.0
+
+  for (i in 0 to n, l in row-locs(W,n,3.81)) do :
```

Comment:
`l` is pretty had to find in a code. 
For iterators like that, I like to suggest to double the letter: `ii` or `ll`

Reply:
`loc` is fine too here.